docs/octant-plugin-installation.md

```diff
@@ -91,25 +91,38 @@ You can follow the steps listed below to install octant and antrea-octant-plugin
     export KUBECONFIG=/etc/kubernetes/admin.conf
     ```
 
-3. Build antrea-octant-plugin.
+3. Get corresponding antrea-octant-plugin binary from [Release Assets](https://github.com/vmware-tanzu/antrea/releases/tag/v0.8.1)
+based on your environment and move the binary to OCTANT_PLUGIN_PATH.
 
-    ```
-    cd plugins/octant
-    make antrea-octant-plugin
-    ```
-
-4. Move antrea-octant-plugin to OCTANT_PLUGIN_PATH.
+    For example, you can get antrea-octant-plugin-linux-x86_64 if it matches your operating system and architecture.
 
     ```
```

Comment:
add `bash` after the triple backticks, I think Cody mentioned something about this being needed for the website (same comment for below)

Reply:
Done.